app/src/main/java/com/example/weatherapp/FragmentOne.java

```diff
@@ -0,0 +1,64 @@
+package com.example.weatherapp;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link FragmentOne#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class FragmentOne extends Fragment {
```

Comment:
Rewrite this in Kotlin, give a more meaningful name and delete the unnecessary boilerplate code